pystac/stac_object.py

```diff
@@ -81,21 +81,40 @@ def remove_links(self, rel: Union[str, pystac.RelType]) -> None:
 
         self.links = [link for link in self.links if link.rel != rel]
 
-    def get_single_link(self, rel: Union[str, pystac.RelType]) -> Optional[Link]:
+    def get_single_link(
+        self,
+        rel: Union[str, pystac.RelType],
+        media_type: Optional[Union[str, pystac.MediaType]] = None,
+    ) -> Optional[Link]:
         """Get single link that match the given ``rel``.
```

Comment:
Sorry I didn't catch this earlier, but we should update the docstring here as well.
```suggestion
        """Get single link that match the given ``rel`` and, optionally,
        ``media_type``. If ``media_type`` is ``None``, then the link is
        matched only on the ``rel`` value.
```